### ISSUE TITLE
Recommend official JARs for Linux/Mac

### ISSUE THREAD

**Issue by superbobry** [criteo-forks/xgboost-jars#5]
Now that xgboost4j releases its own JARs we should explicitly recommend them in the README and mention that this project exists solely for Windows builds.

**Comment by trams** [comment]
We can update README. But in case we recommend to use official JARs for every platform aside from Windows shouldn't we remove Travis CI integration?

**Comment by trams** [comment]
@superbobry, a friendly ping

**Comment by superbobry** [comment]
Yep, Travis can be dropped. I think a proper "fix" for this is to contribute Azure Pipelines integration upstream and cover all three platforms.

**Comment by trams** [comment]
I thought about a similar plan. That we should contribute this repo upstream

**Comment by haohenggang** [comment]
Is there any xgboost4j.jar available for Android phones that we would like to train online on?
